src/components/shared/Fields/Form/ActionHookForm.tsx

```diff
@@ -39,19 +43,23 @@ interface Props<V extends Record<string, any>>
 }
 
 const ActionHookForm = <V extends Record<string, any>>({
-  error,
+  actionType,
   onSuccess,
   onError,
   onSubmitError,
+  transform,
   submit,
+  error,
   success,
-  transform,
   ...props
 }: Props<V>) => {
+  const submitAction = submit || actionType;
```

Comment:
Updating `ActionHookForm` to share same api as `ActionButton` re use of actionType to infer success/error action types